lms/static/scripts/frontend_apps/components/OAuth2RedirectErrorApp.js

```diff
@@ -93,7 +105,26 @@ export default function OAuth2RedirectErrorApp({ location = window.location }) {
           </p>
         </Fragment>
       )}
-      <ErrorDisplay message={message} error={error} />
+
+      {error.code === 'blackboard_missing_integration' && (
+        <Fragment>
+          <p>
+            A blackboard admin needs to add or enable a REST API integration.
+          </p>
+          <p>
+            For more information see:{' '}
+            <a
+              target="_blank"
+              rel="noopener noreferrer"
+              href="https://github.com/hypothesis/lms/wiki/How-to-set-up-Blackboard-API-access-for-the-LMS-app#create-a-rest-api-integration-in-the-blackboard-site"
+            >
+              Create a REST API Integration in the Blackboard site
+            </a>
+            .
+          </p>
+        </Fragment>
+      )}
+      <ErrorDisplay message={message()} error={error} />
```

Comment:
This shows both the concat support and a (I think) superfluous other message which here I temporarly replaced with "."

![Screenshot from 2021-08-11 15-53-45](https://user-images.githubusercontent.com/1433832/129041822-2fae7aca-c52d-43fd-9e45-c53d8f2b5709.png)

Reply:
For the purposes of this PR, I suggest we just make this Blackboard error's display match the existing one for Canvas, which currently has a "Something went wrong authorizing Hypothesis" message.

If we wanted to remove the message in future we could make it so that `<ErrorDisplay message={null} .../>` is treated differently from `<ErrorDisplay .../>` without a `message` prop. I forget where, but we do have one other place that uses this pattern - where you set the prop to a value to use, set it to `null` to  not use the feature or omit it to get a default value.